src/HasTags.php

```diff
@@ -19,6 +19,16 @@ public static function getTagClassName(): string
         return config('tags.tag_model', Tag::class);
     }
 
+    public static function getTaggableMorphName(): string
```

Comment:
Make these non-static

Reply:
Should I just make `getTaggableMorphName` non-static?
or with `getTaggableTableName` ?

Reply:
Let's make both non-static 👍

Reply:
done!